cogs/images.py

```diff
@@ -54,17 +54,18 @@ class Images(commands.Cog):
 
         EXAMPLE: !snek
         RESULT: A beautiful picture of a snek o3o"""
-        result = await utils.request("http://hrsendl.com/snake")
+        data = await utils.request("http://hrsendl.com/api/snake")
+        result = data['data']
```

Comment:
This should be done after the if statement, and the if statement should be changed to check `data` since that's checking if there was some network issue and it couldn't connect at all.